contracts/HashRegistrarSimplified.sol

```diff
@@ -381,7 +416,7 @@ contract Registrar {
         Deed bid = sealedBids[bidder][seal];
         // If the bid hasn't been revealed after any possible auction date, then close it
         if (address(bid) == 0
-            || now < bid.creationDate() + initialAuctionPeriod) throw;
+            || now < bid.creationDate() + launchLength) throw;
```

Comment:
There's an edge case here: if I bid on an auction and do not reveal, but neither does anyone else, you can still reopen the auction reveal again and not lose the deposit. It's unintentional and I imagine could be useful for people who are trying to buy obscure names.

Also, if for some reason you create a bid before opening an auction, it means that for a short period of time, your bid could be invalidated.

Finally, if you forgot to bid, revealing it after the launch will get your extra deposit back (and reveal some more data to the market) so it's always better for users to reveal invalid bids than cancelling them and making the cancel period short, it creates a race.

It seems to me increasing the time for cancelling a bid would be tolerant of those mistakes and still allow cleaning of transactions on the future.